libpod/define/pod_inspect.go

```diff
@@ -103,6 +105,8 @@ type InspectPodInfraConfig struct {
 	CPUQuota int64 `json:"cpu_quota,omitempty"`
 	// CPUSetCPUs contains linux specific CPU data for the container
 	CPUSetCPUs string `json:"cpuset_cpus,omitempty"`
+	// Pid contains the process id namespace information for the container
+	Pid string `json:"pid,omitempty"`
```

Comment:
This only needs to be in one place - I think here is probably the more sensible one